docs/README.md

```diff
@@ -144,13 +144,15 @@ ecu_info.yaml is the setting file for ECU configuration.
 
 - available_ecu_ids (optional)
 
-  This field specifies a list of all ECU ids, including directly connected, indirectly connected and itself.
+  This field specifies a list of all ECU ids to be updated.  
   Only the main ECU should have this information.  
-  If this field is not specified, value of `ecu_id` is used as this value.  
+  For compatibility, if this `available_ecu_ids` filed does not exist, the ecu_id of the main ecu is considered the entry for available_ecu_ids.
```

Comment:
I think this line should also prefixed with **NOTE**(or even **WARNING**), as user might think empty `available_ecu_ids` means not update any ECUs.
Also, I think this is `backward-compatibility`?